external/vcm/tox_install.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+set -e -x
+
+pip install -c ../../constraints.txt numpy
```

Comment:
tox really doesn't want to do multiple install commands, which meant I had to use this script. For some reason it also switched to doing a "nodeps" install of vcm, which necessitated the final `pip install .`. Took about an hour of trial-and-error to get this right.